demo/summer-2016.html

```diff
@@ -240,23 +240,13 @@
     });
 
 
+    require(["ace/autocomplete/papeeria/tex_completer"], function(completer) {
```

Comment:
Unless this prefix `lib/ace/mode` has some special meaning to ace, I'd prefer to have all our stuff under the same folder. I don't think that it will prevent you from modifying papeeria_latex.js if you need it :)